app/views/caseload.html

```diff
@@ -165,6 +184,22 @@ <h3 class="heading-medium">{{ caseType.displayName | title }} caseload by tier</
               </tr>
             {% endfor %}
             {% endfor %}
+            <tr>
+                <td rowspan="2" headers="{{caseType.displayName}}_om {{caseType.displayName}}_name">Total</td>
```

Comment:
I think the rowspan of these will be determined by the number of grades so it should probably by dynamic e.g. `caseType.array.totals.length`.

Reply:
An even better shout, will get that in too.